extensions/circuit/src/eq/core.rs

```diff
@@ -3,7 +3,10 @@ use std::{
     borrow::{Borrow, BorrowMut},
 };
```

Comment:
Unfortunately we can't remove `eq core` because the original chip in OpenVM is `branch_eq`, not just `eq`.